test/jdk/jdk/dynalink/TrustedDynamicLinkerFactoryTest.java

```diff
@@ -193,7 +193,7 @@ public void prelinkTransformerTest() throws Throwable {
         final CallSite cs = linker.link(new SimpleRelinkableCallSite(new CallSiteDescriptor(
                 MethodHandles.publicLookup(), GET_PROPERTY, mt)));
         Assert.assertFalse(reachedPrelinkTransformer[0]);
-        Assert.assertEquals(cs.getTarget().invoke(new Object(), "class"), Object.class);
+        Assert.assertEquals(cs.getTarget().invoke(new Object(), "class"), java.util.Objects.newIdentity().getClass());
```

Comment:
Similar to BeanLinkerTest.java, it's clearer to define a TestObject class to replace "new Object()" in this test.

Reply:
Done!